main.go

```diff
@@ -238,7 +254,8 @@ func main() {
 		os.Exit(0)
 	}
 
-	if len(os.Args) < 4 {
+	// every command needs at least two parameters - the command to run and the trace ID
```

Comment:
should the individual commands have an internal less-than check?

Reply:
I want to rewrite the whole arg parsing thing - individual commands shouldn't be looking to `os` to get their arguments. They're functions; they should be function arguments. I put off any of those changes in favor of doing a better job later.